util/election/tracker.go

```diff
@@ -69,28 +69,28 @@ func (mt *MasterTracker) Count() int {
 }
 
 // Held returns a (sorted) list of the IDs for which we are currently master.
-func (mt *MasterTracker) Held() []int64 {
+func (mt *MasterTracker) Held() []string {
 	mt.mu.RLock()
 	defer mt.mu.RUnlock()
-	ids := make([]int64, 0, mt.masterCount)
+	ids := make([]string, 0, mt.masterCount)
 	for id := range mt.masterFor {
 		if mt.masterFor[id] {
 			ids = append(ids, id)
 		}
 	}
-	sort.Sort(int64arr(ids))
+	sort.Strings(ids)
 	return ids
 }
 
 // IDs returns a (sorted) list of the IDs that we are currently tracking.
-func (mt *MasterTracker) IDs() []int64 {
+func (mt *MasterTracker) IDs() []string {
 	mt.mu.RLock()
 	defer mt.mu.RUnlock()
-	ids := make([]int64, 0, len(mt.masterFor))
+	ids := make([]string, 0, len(mt.masterFor))
 	for id := range mt.masterFor {
 		ids = append(ids, id)
 	}
-	sort.Sort(int64arr(ids))
+	sort.Strings(ids)
```

Comment:
Are we okay to switch the sorting order from digital to lexicographical? Could it mess things in monitoring etc?

Reply:
Good catch.  @daviddrysdale ? 
To my knowledge this shouldn't cause any issues.